AdobeMediaGallery/Test/Unit/Model/Asset/Command/DeleteByIdTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AdobeMediaGallery\Test\Unit\Model\Asset\Command;
+
+use Magento\AdobeMediaGallery\Model\Asset\Command\DeleteById;
+use Magento\AdobeMediaGalleryApi\Api\Data\AssetInterface;
+use Magento\AdobeMediaGalleryApi\Api\Data\AssetInterfaceFactory;
+use Magento\AdobeMediaGalleryApi\Model\Asset\Command\GetByIdInterface;
+use Magento\Framework\App\ResourceConnection;
+use Magento\Framework\DB\Adapter\AdapterInterface;
+use Magento\Framework\Exception\CouldNotDeleteException;
+use PHPUnit\Framework\TestCase;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Test the DeleteById command model
+ */
+class DeleteByIdTest extends TestCase
+{
+    /**
+     * @var DeleteById
+     */
+    private $deleteMediaAssetById;
+
+    /**
+     * @var ResourceConnection
+     */
+    private $resourceConnection;
+
+    /**
+     * @var GetByIdInterface
+     */
+    private $getMediaAssetById;
+
+    /**
+     * @var AssetInterfaceFactory
+     */
+    private $mediaAssetFactory;
+
+    /**
+     * @var AssetInterface
+     */
+    private $mediaAsset;
+
+    /**
+     * @var AdapterInterface
+     */
+    private $adapter;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * Initialize basic test class mocks
+     */
+    protected function setUp()
+    {
+        $this->resourceConnection = $this->createMock(ResourceConnection::class);
+        $this->getMediaAssetById = $this->createMock(GetByIdInterface::class);
+        $this->mediaAssetFactory = $this->createMock(AssetInterfaceFactory::class);
+        $this->logger = $this->createMock(LoggerInterface::class);
+
+        $this->deleteMediaAssetById = (new ObjectManager($this))->getObject(
+            DeleteById::class,
+            [
+                'resourceConnection' => $this->resourceConnection,
+                'getMediaAssetById' => $this->getMediaAssetById,
+                'mediaAssetFactory' =>  $this->mediaAssetFactory,
+                'logger' =>  $this->logger,
+            ]
+        );
+
+        $this->mediaAsset = $this->createMock(AssetInterface::class);
+        $this->adapter = $this->createMock(AdapterInterface::class);
```

Comment:
If the `adaptor` is created in `setUp` - it's a good idea to configure the `resourceConnection` mock to return this adaptor in `setUp` method as well.

In this case we don't need a property for `resourceConnection` and will work only with `adaptor` property further